specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/1.2-preview/iotcentral.json

```diff
@@ -0,0 +1,5641 @@
+          "readOnly": true
+        },
+        "@type": {
+          "description": "The JSON-LD types of this device template.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "etag": {
+          "description": "ETag used to prevent conflict in device template updates.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Display name of the device template.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Detailed description of the device template.",
+          "type": "string"
+        },
+        "capabilityModel": {
+          "description": "The capability model utilized by this device template.",
+          "type": "object"
+        },
+        "deploymentManifest": {
+          "description": "Deployment manifest associated to this device template.",
+          "type": "object"
+        }
+      },
+      "required": [
+        "@type",
+        "capabilityModel"
+      ]
+    },
+    "DeviceTemplateCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection of device templates.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeviceTemplate"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page of device templates.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DeviceTemplateMigrationJobData": {
+      "x-ms-discriminator-value": "deviceTemplateMigration",
+      "allOf": [
+        {
+          "$ref": "#/definitions/JobData"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "template": {
+          "description": "The target device template to which devices will be migrated.",
+          "type": "string",
+          "minLength": 1
+        }
+      },
+      "required": [
+        "template"
+      ]
+    },
+    "EmailUser": {
+      "x-ms-discriminator-value": "email",
+      "allOf": [
+        {
+          "$ref": "#/definitions/User"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "email": {
+          "description": "Email address of the user.",
+          "type": "string",
+          "format": "email"
+        }
+      },
+      "required": [
+        "email"
+      ]
+    },
+    "Enrichment": {
+      "type": "object",
+      "properties": {
+        "target": {
+          "description": "The device template or interface which defines the target capability for the enrichment.",
+          "type": "string"
+        },
+        "path": {
+          "description": "The path to the target capability within the device template or the system property to use.",
+          "type": "string"
+        },
+        "value": {
+          "description": "The raw value used for the enrichment."
```

Comment:
Could this really be any type?  An int or a string or float?

Reply:
It's any type, within OPENAPI 2.0 there is way to specify any type as 3.0, leave it blank actually ensured auto rest generated client use property type.